clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/models/AzureLoadBalancerDescription.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.azure.client.models
+
+class AzureLoadBalancerDescription extends AzureBaseDescription {
+  String loadBalancerName
+  String stack
+  String detail
+  String vnet
+  List<AzureLoadBalancerProbe> probes
```

Comment:
List<AzureLoadBalancerProbes> probes = []

Would avoid the need for an initializing constructor below